docs/reference/src/advanced/backend_linuxkms.md

```diff
@@ -34,6 +34,7 @@ The LinuxKMS backend supports different renderers. They can be explicitly select
 |---------------|------------------------|-----------------------------------------------------------------------------|
 | FemtoVG       | OpenGL ES 2.0          | `linuxkms-femtovg`                                                          |
 | Skia          | OpenGL ES 2.0, Vulkan  | `linuxkms-skia-opengl`, `linuxkms-skia-vulkan`, or `linuxkms-skia-software` |
+| software      | None                   | `linuxkms-software`                                                         |
```

Comment:
Should the missing cursor be documented, perhaps?

Reply:
Good idea, I'll add that.